routers/api/v1/utils/hook.go

```diff
@@ -5,14 +5,16 @@
 package utils
 
 import (
-	api "code.gitea.io/sdk/gitea"
-
 	"encoding/json"
 	"net/http"
+	"strings"
+
+	api "code.gitea.io/sdk/gitea"
 
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/context"
 	"code.gitea.io/gitea/routers/api/v1/convert"
+	"code.gitea.io/gitea/routers/utils"
```

Comment:
add an empty line between internal and external package.

Reply:
Done @appleboy 968d1f2274dc11ab73f0e670d68a0a56a950f238 .. Not sure why this didn't get collapsed